final_prep/mod_zero_hero.rb

```diff
@@ -1,84 +1,99 @@
-#assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
-#assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
-# Declare a new variable - scary_monster - assigned to an hash with the following key/values
-#   - name (string)
-#   - smell (string)
-#   - weight (integer)
-#   - citiesDestroyed (array)
-#   - luckyNumbers (array)
-#   - address (hash with following key/values: number , street , state, zip)
-
-
-# Create a new class called SuperHero
-# - Your class should have the following DYNAMIC values
-#   - name 
-#   - super_power
-#   - age 
-# - Your class should have the following STATIC values
-#   - arch_nemesis, assigned to "The Syntax Error"
-#   - power_level = 100
-#   - energy_level = 50 
-
-# - Create the following class methods
-#   - say_name, should print the hero's name to the terminal
-#   - maximize_energy, should update the energy_level to 1000
-#   - gain_power, should take an argument of a number and INCREASE the power_level by that number
-
-# - Create 2 instances of your SuperHero class
-
-
-# Reflection
+hero_name = "Radioactive Man"
+special_ability = "upending atoms"
+greeting = "Behold! I am #{hero_name}."
+catchphrase = "Up and at them, #{special_ability}!"
+power = 3
+energy = 7
+full_power = power * 500
+full_energy = energy + 150
+is_human = true
+identity_concealed = false
+arch_enemies = ["King Kong", "James Joyce", "Studio Executive"]
+sidekicks = ["Fallout Boy", "Jolly Old St Nick", "Mickey Rooney"]
+arch_enemies.push("Johnny Rocket")
+puts arch_enemies
+sidekicks.shift
+puts sidekicks
+
+def asses_situation(danger_level, save_the_day = "Praise me, the almightly atomized Radioactive Man!", bad_excuse = "These atoms are too dense!")
+
+  puts "The danger level appears to be #{danger_level}!"
+if danger_level >= 50
+  puts bad_excuse
+
+elsif danger_level < 50 and danger_level > 10
+  puts save_the_day
+
+elsif danger_level < 10
+  puts "Meh. Hard Pass"
+end
+end
+puts asses_situation(50)
+
+scary_monster = {
+  "name" => "The Three Mile Islander",
+  "smell" => "floral",
+  "weight" => 150,
+  "cities_destroyed" => ["Cleveland," "San Fran", "Pueblo"],
+  "lucky_numbers" => [7, 77, 777],
+  "address" => {
+  "number" => 19072,
+  "street" => "Jason St",
+  "state" => "CO",
+  "zip" => 80211,
+  }
+}
+class SuperHero
+  attr_accessor :nemesis, :super_power, :age , :nemesis , :energy_level, :power_level, :name
+  def initialize(nemesis, power_level, energy_level)
+    @nemesis = nemesis
+    @energy_level = energy_level
+    @power_level = power_level
+  end
+
+  def say_name
+    puts name
+  end
+
+  def maximize_energy
+    energy_level = 1000
+    puts energy_level
+  end
+
+  def gain_power(power_gain)
+    puts "POWER GAIN!!!"
+   puts "#{power_gain} + #{power_level} = NEW POWER LEVEL!"
+  end
+
+end
+
+Radioactiveman = SuperHero.new("The Syntax Error", "100", "50")
+Radioactiveman.name = "Radioactive Man"
```

Comment:
Ah- I see you assign your class a name property here, but it is typically best practice to have this in your properties, so other devs can look at the class, and see all of it right there.